### PR TITLE
Clarify that npm@2 is required for scoped packages

### PR DESCRIPTION
**Pull request by danpaz**
Resolves https://github.com/npm/docs/issues/417.

Makes it a little clearer that scoped packages will only work with `npm@2` or higher.

### CODE REVIEW

**Comment by othiym23** [comment]
I think that in both cases it deserves to be two sentences. Also, the fact that it's supported by the primary npm registry should probably come first. At some point we're probably going to clean up the docs to have a better concept of when various features were introduced, but this is a good change for now. Thanks for getting this started!

**Comment by ashleygwilliams** [comment]
hey! this will all get much easier when i finally get the docs versioned. mind if i take the reins on this PR to make the language a little more newb friendly? this is a super great start :sparkling_heart: :dancers:

**Comment by danpaz** [comment]
@ashleygwilliams Go for it! :shipit:

**Comment by othiym23** [comment]
I'm not sure where versioned npm documentation is on @ashleygwilliams's road map, but there's no reason why (with a few tweaks) this can't be merged now, to both LTS and latest.

File: doc/misc/npm-scope.md [hunk 1]
```diff
@@ -51,8 +52,9 @@ just specifying to require the module `mypackage` in the folder called `@myorg`.
 
 ## Publishing scoped packages
 
-Scoped packages can be published to any registry that supports them, including
-the public npm registry.
+Scoped packages can be published from the cli as of `npm@2` and can be
```
Comment (othiym23) [inline doc/misc/npm-scope.md]: "cli" should be either "CLI" or "npm client".

File: doc/misc/npm-scope.md [hunk 2]
```diff
@@ -13,9 +13,10 @@ followed by a slash, e.g.
 Scopes are a way of grouping related packages together, and also affect a few
 things about the way npm treats the package.
 
-Scoped packages are supported by the public npm registry.  The npm
-client is backwards-compatible with un-scoped registries, so it can be
-used to work with scoped and un-scoped registries at the same time.
+Scoped packages can be published and installed as of `npm@2` and are supported
+by the public npm registry. The npm client is backwards-compatible with
```
Comment (othiym23) [inline doc/misc/npm-scope.md]: My current preferred term of art is "primary npm registry".

File: doc/misc/npm-scope.md [hunk 3]
```diff
@@ -51,8 +52,9 @@ just specifying to require the module `mypackage` in the folder called `@myorg`.
 
 ## Publishing scoped packages
 
-Scoped packages can be published to any registry that supports them, including
-the public npm registry.
+Scoped packages can be published from the cli as of `npm@2` and can be
+published to any registry that supports them, including the public npm
```
Comment (othiym23) [inline doc/misc/npm-scope.md]: "primary npm registry", as above.

**Comment by othiym23** [comment]
Either @danpaz can clean up those stylistic patches, or whomever lands this can do so. This is ready to be merged!

**Comment by danpaz** [comment]
@othiym23 @ashleygwilliams Thanks! Updated and merged master.

**Comment by iarna** [comment]
This is merged and will be in the next release (scheduled for later today).

**Pull request closed without merging**